lib/plugins/aws/provider/awsProvider.js

```diff
@@ -146,22 +146,11 @@ class AwsProvider {
         f()
           .then(resolve)
           .catch((e) => {
-            const err = e;
-            if (err.statusCode === 429) {
+            if (e.statusCode === 429) {
```

Comment:
Can we quick-fix this bug in a separate PR since it's unrelated to the shared API Gateway PR (I can do that since I broke it recently 😬).